variables.container.instance.tf

```diff
@@ -0,0 +1,60 @@
+variable "container_instance_count" {
+  type        = number
+  description = "The number of container instances to create"
+  default     = 2
+}
+
+variable "container_instance_name_prefix" {
+  type        = string
+  description = "The name prefix of the container instance"
+  default     = null
+}
+
+variable "container_instance_container_name" {
+  type        = string
+  description = "The name of the container instance"
+  default     = null
+}
+
+variable "container_instance_environment_variables" {
+  type = set(object({
```

Comment:
Could this be a map(string) since the key has to be unique?

Reply:
Container Apps has them in this format instead of a map, which is why it is like this. It is also possible that someone may want to supply an env var kay as an output from another resource. Edge case I know, but would result in a not know at apply time error.

Reply:
Cool, np